config/routes.rb

```diff
@@ -68,9 +68,10 @@
       put  :proposition
       get  :indicateurs, on: :collection
       post '/payment_registry', to: 'payment_registries#create'
-      resources :payments, only: [:new, :create, :edit, :update, :destroy, :ask_for_validation, :ask_for_modification], param: :payment_id do
+      resources :payments, only: [:new, :create, :edit, :update, :destroy, :ask_for_validation, :ask_for_modification, :send_in_opal], param: :payment_id do
```

Comment:
Si j'ai bien compris cela permet d'avoir params[:payment_id] instead of params[:id]